static/bidder-params/triplelift.json

```diff
@@ -11,11 +11,5 @@
     },
     "floor" :  {"description" : "the bid floor", "type": "number" }
   },
-  "oneOf": [{
-    "oneOf": [{
-      "required": ["inventoryCode"]
-    }] }],
-  "not": {
-    "required": ["floor"]
-  }
+  "required": ["inventoryCode"]
```

Comment:
Is the ask to make "floor" required as well?

Reply:
No, only `Inventory Code` is required